internal/ent/schema/mappedcontrol.go

```diff
@@ -0,0 +1,108 @@
+package schema
+
+import (
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent"
+	"entgo.io/ent/schema"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/index"
+
+	"github.com/theopenlane/core/internal/ent/generated/privacy"
+	"github.com/theopenlane/core/internal/ent/mixin"
+	"github.com/theopenlane/core/internal/ent/privacy/policy"
+	emixin "github.com/theopenlane/entx/mixin"
+)
+
+// MappedControl holds the schema definition for the MappedControl entity
+type MappedControl struct {
+	ent.Schema
+}
+
+// Fields of the MappedControl
+func (MappedControl) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("control_id").
+			Comment("the id of the control being mapped").
+			Unique().
+			Immutable().
+			NotEmpty(),
+		field.String("mapped_control_id").
+			Unique().
+			Immutable().
```

Comment:
same comment with this one, although the reason is purely the re-use of code combinations